tests/integration/kubernetes/k8s-block-volume.bats

```diff
@@ -10,7 +10,13 @@ load "${BATS_TEST_DIRNAME}/../../common.bash"
 load "${BATS_TEST_DIRNAME}/tests_common.sh"
 
 setup() {
-	[ "${KATA_HYPERVISOR}" == "qemu-runtime-rs" ] && skip "See: https://github.com/kata-containers/kata-containers/issues/10373"
+	 case "${KATA_HYPERVISOR}" in
+	 	qemu-runtime-rs)
```

Comment:
Hi @stevenhorsman , yeah, I noticed those failures as well. But I want to caught your attention to this:

10373
10873

Exactly 500 issues later, same test failing! It's a signal. I will bet on the lottery for sure, don't loose that opportunity and do the same!